src/main/java/core/basesyntax/service/TransactionProcessor.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.Map;
+
+public class TransactionProcessor {
+    private Map<String, Integer> fruits;
+
+    public void process(List<FruitTransaction> transactions) {
+        for (FruitTransaction transaction : transactions) {
+            String fruit = transaction.fruit();
+            int quantity = transaction.quantity();
+            switch (transaction.operation()) {
```

Comment:
Check archirectural diagram in task description, you should have OperationStrategy